### ISSUE TITLE
Place of publication (RDA element) vs. place of publication (place name) and similar PPDM elements

### ISSUE THREAD

**Issue by jess-grz** [rbms-bsc/DCRMR#148]
During various stages of review, we have gone back and forth in [chapter 5](https://bsc.rbms.info/DCRMR/ppdm/) (PPDM) on when to refer to the element proper (e.g., Place of publication) and to the term more generally (e.g., place of publication in the sense of a place name rather than the value of the element). We should take a more holistic look at how we're referring to these elements and concepts throughout chapter 5 and bring clarity and consistency to the text as needed.

**Comment by ladylazarus3** [comment]
Chapter 5 has been an interesting journey ("interesting" used here in the true Minnesotan sense).  i would say that if we were to take a more holistic look at how we are referring to these elements and concepts throughout chapter 5, that this focus should be to cleave closer to our principles as outlined in the [Readme.md](https://github.com/rbms-bsc/DCRMR#current-status) and [Integrity of the DCRM text](https://github.com/rbms-bsc/DCRMR/wiki/Integrity-of-DCRM-text) in that "Language in the revised manual will be heavily based on the current manuals."  and "Do not change wording in future modules unless it clarifies an ambiguity, corrects a mistake, or needs changes for material-specific (or, perhaps in dcrmr's case, *exasperated sigh* "oh RDA!") reasons."

On a more practical and somewhat completely selfish note, take the example of the following instruction in chapter 5:
`If both the place and [Name of publisher](https://bsc.rbms.info/DCRMR/ppdm/Name-of-publisher/) associated with an earlier...`  

which takes a lot more effort on the behalf of current and future keepers of the text in terms of link maintenance and keying in urls correctly than does the exceedingly clear DCRM(B) instruction it was derived from, 

`If both the place and publisher ... associated with an earlier...`

in terms of sustainability of the text and the labor involved.  For these reasons, i would be a fan of fewer element references than more element references included within the text, but other folks may feel differently.  

kalan

**Comment by jess-grz** [comment]
Those are excellent points, @ladylazarus3!

As background for opening this issue, some group members pointed out in the summer edits doc that we should consider using the formal element name rather than the more generic term in the instructions for various PPDM elements in chapter 5. I'm remaining neutral on the topic for now (since I've spent a little too much time with chapter 5!). Especially since we have several new group members now, it seemed like a good time to have others weigh in if they'd like.

In any case, I don't think we'd make any changes related to this issue in chapter 5 in the near future, unless they are already being addressed in the graphics draft.

**Comment by ladylazarus3** [comment]
Totally, i hear that! <3  i also feel like i've spent way too much time in chapter 5 and definitely want to hear fresh takes as i've lost a lot of objectivity in the matter.  If folks feel the links and formal element names (e.g., `[Name of publisher](/DCRMR/ppdm/Name-of-publisher/)` rather than `publisher` ) make the instructions clearer in terms of RDA, then it would make sense to use them.

**Comment by eblakedc** [comment]
Does the formal element name always have to have a link? I like consistent use of formal element names because it helps me acclimatize to RDA 3R: if there's a capital letter at the start of the phrase, I know there's an RDA element to go with it. Having lots of them be links, though, can be distracting (not to mention harder to maintain, as @ladylazarus3 points out). 

There are times it _is_ helpful to be able to jump from one section to a specific place in another, but linking to the element name's heading doesn't add much value, especially because the document's design makes it easy to find those headings anyway (hoooray!).  

That is to say, no matter where I am in the document, I can easily skim the left-hand side bar for the Name of thing that is capitalized. For example, I'd expect both "Place of publication" and "Name of publisher" to be capitalized in 5.22.31.3 (because they're names of elements) but I don't need them to jump out as links. I'm already in the PPDM part of the instructions, and the headings I'd need if I wanted clarification are easy find in the sidebar. 
![image](https://user-images.githubusercontent.com/107492238/194635638-8425d895-d3c6-41b0-9020-ec894a461fe4.png)

Another example: In 5.22.32.1, it's important for "see **5.24.34.1**" to be big and linky because it'll take me to the exact place I need to be to figure out what to do next. But it's not important for "Note on publication statement" to also be a link. If I want to learn about the element itself, I can go there via the side-bar. 
![image](https://user-images.githubusercontent.com/107492238/194637193-ea3663d2-1854-467a-bf7e-abcaad763dac.png)

**Comment by rarebkcat** [comment]
Two notes in response to @eblakedc:

1. Formal element names are always capitalized and linkified. The exception is if they refer to the element that you're in -- if you're in, e.g., Title proper, we capitalize but don't link to Title proper because that would be incredibly circular. The [Style sheet (full) section on internal links](https://github.com/rbms-bsc/DCRMR/wiki/Style-sheet-(full)#links-internal) currently says "Capitalize element names and hyperlink to element pages; do not precede element names with the RDA entity."

* Where this becomes messy is: when is it a formal element name (RDA element: Place of publication; RDA element: Title proper; RDA element: Identifier for manifestation) and when is it the thing being recorded (the place name; the text on the preferred source of information; the ISBN)? 

2. The navigation sidebar is delightful! However, if you're in mobile view (or on a computer with a narrower screen), the screen collapses and then the navigation is a toggle at the top. So, depending on your interface, you can't always scroll and click from the sidebar because the sidebar might not exist. 

I was going to then write a note that was not in direct response to @eblakedc, but is general on the topic of elements, information, and information-in-elements, but:

@jess-grz mentions spending too much time with Chapter 5, and I'll go ahead second that sentiment. 

Can an element be a grammatically inseparable part of another element? This comes up with other title information and title proper, as well as place of publication and publisher, for example. (Lower case deliberately used.) Do we always just want to link to our pages for an element instead of trying to parse it? Can you separate an element from its content, or is the element the content itself -- does the RDA element: Place of publication have the content/value of the place name, or *is* the place name synonymous with RDA element: Place of publication? 

I'll leave it at: I don't even know.

**Comment by eblakedc** [comment]
Thanks @rarebkcat! This is exactly what I needed to see, and I really appreciate that you took the time to provide the links so that I could bring myself up to speed.

**Comment by eblakedc** [comment]
It would be great if we could come up with a principle for sorting out the messiness that @rarebkcat refers to here:

> Where this becomes messy is: when is it a formal element name (RDA element: Place of publication; RDA element: Title proper; RDA element: Identifier for manifestation) and when is it the thing being recorded (the place name; the text on the preferred source of information; the ISBN)?

For example, would this work?

- When talking about the information itself (that is, what you _will be_ transcribing or supplying) do not capitalize.
- When talking about the element as such (that is, the location _where_ you'll be transcribing or supplying that information) use the formal Element name. 

For 5.22.33.1, that would give us (new text in italics):

> 5.22.33.1 If the <strike>Name of publisher</strike> _publisher's name_ contains grammatically inseparable statements relating to <strike>Place of publication or Date of publication</strike> _the place or date of publication_, transcribe them as part of the Name of <strike>the</strike> publisher. <strike>Supply</strike> _Use square brackets to supply the_ [Place of publication](https://bsc.rbms.info/DCRMR/ppdm/Place-of-publication/) or [Date of publication](https://bsc.rbms.info/DCRMR/ppdm/Date-of-publication/) <strike>in square brackets in the appropriate element</strike>.

**Comment by rarebkcat** [comment]
So, this is where I admit that one way I think through it, when I am contemplating the messy bits and whether what's being referred to is the formal element or the information that goes in the formal element, is:

"If I said '245 subfield $a here' would it make sense, or would I want to specifically say 'title proper'? If the former, then it's the formal element; if the latter, then the information that goes in it." (Or whatever the equivalent is for the element/information in question.)

(Yes, I know, it's MARC. XD But I feel like we do that kind of conflation all the time in the context of talking cataloging, and sometimes we do mean both and sometimes we don't. So it's a way for me to think about what is meant while taking the duplicated terminology out of it.) 

And maybe we don't want to maintain the very-fine-distinction between, e.g., RDA element: Title proper and the title proper, between the formal element and its content/value. (Or, in this instance, RDA element: Place of publication and the place name.) And there's where I am way too close to it all to have any kind of coherent thought or opinion anymore! 

Although, yes, if we do, then a clearly stated principle would be *lovely*!

**Comment by JessieSherwood** [comment]
In the midst of reviewing the notes for Chapter 5, I fervently agree with kalan's assessment that it's interesting (in the most Minnesotan sense).

**Comment by jess-grz** [comment]
@io-uk @eblakedc I agree with you both. Once our work on graphics and policy statements calms down, we may want to consider assigning a small group to comb through all of chapter 5 and make these detailed changes as needed. Of course, the results of this project would appear in a later release, perhaps the one after the graphics release. The type of semantic confusion that we've described in this issue is most pervasive in chapter 5, so it would be a good case study.
